signingscript/docker.d/passwords.yml

```diff
@@ -271,30 +271,12 @@ in:
              {"$eval": "AUTOGRAPH_FOCUS_PASSWORD"},
              ["autograph_focus"]
             ]
-        project:mobile:fenix:releng:signing:cert:nightly-signing:
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENIX_NIGHTLY_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENIX_NIGHTLY_PASSWORD"},
-             ["autograph_apk"]
-            ]
-        project:mobile:fenix:releng:signing:cert:beta-signing:
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENIX_BETA_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENIX_BETA_PASSWORD"},
-             ["autograph_apk"]
-            ]
         project:mobile:fenix:releng:signing:cert:production-signing:
           - ["https://autograph-external.prod.autograph.services.mozaws.net",
              {"$eval": "AUTOGRAPH_FENIX_USERNAME"},
              {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
              ["autograph_apk"]
             ]
-        project:mobile:fenix:releng:signing:cert:fennec-nightly-signing:
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENNEC_NIGHTLY_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENNEC_NIGHTLY_PASSWORD"},
-             ["autograph_apk_fennec_sha1"]
-            ]
         project:mobile:fenix:releng:signing:cert:fennec-production-signing:
```

Comment:
I reckon we still need until the migration is over, right?

Reply:
That's right!